values.go

```diff
@@ -33,6 +36,7 @@ type Value interface {
 	isValue()
 	Type() Type
 	ToGoValue() interface{}
+	String() string //not really sure if this is needed?
```

Comment:
We can re-use the existing standard interface `fmt.Stringer` here:

```suggestion
	fmt.Stringer
```